### DISCUSSION TITLE
Vercel deployment failure

### DISCUSSION THREAD

**Discussion by mikeycrystal** [vercel/next.js#42696] [Help]
I'm trying to deploy the next.js-blog to vercel as part of the starter tutorial. When trying, it gave me the following error in the build portion. In development, npm run dev doesn't work to deploy locally anymore either, where it did throughout the entire development stage. Any ideas?

<img width="783" alt="Screenshot 2022-11-09 at 10 48 48 AM" src="https://user-images.githubusercontent.com/108841529/200890705-b8750483-05ef-4d00-ab6b-0d8bb4ef6410.png">

My repository is: https://github.com/mikeycrystal/nextjs-blog

**Comment by dev-afzalansari** [comment]
@mikeycrystal

You should **Relatively Import** your css modules.

For example see [adding css](https://nextjs.org/docs/basic-features/built-in-css-support)

**Comment by icyJoseph** [comment]
Hi,

The issue is with these lines (and any similar you might have in your codebase, I haven't checked around):

```js
import styles from "/Users/michaelcrystal/Desktop/nextjs-blog/components/layout.module.css";
import utilStyles from "/Users/michaelcrystal/Desktop/nextjs-blog/styles/utils.module.css";
```

These are pointing to your local machine file system, which when deployed to another machine, fails.

Do:

```js
import styles from "./layout.module.css";
import utilStyles from "../styles/utils.module.css";
```

Later one when you learn about `absolute` imports you could shorten the above, but since you are in a learning phase I don't want to overwhelm you.
